src/components/formFields/select/select.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import * as styles from "./select.module.css";
+import { Control, Controller, FieldValues } from "react-hook-form";
+import ReactSelect from "react-select";
```

Comment:
This component has `react-select` as a dependency, I don't see it added to the `package.json`. Because it's missing, the build wont work (and neither will the component).